benchmarks/run-benchmark.sh

```diff
@@ -3,12 +3,47 @@
 # Runs a benchmark using drun. The script assumes drun is available on the caller's path.
 set -euo pipefail
 
+# Remove downloaded didc, drun.
+trap 'set +e && rm drun didc && set -e' EXIT SIGINT
+
 BENCH_NAME=$1
 FILE=$(mktemp)
 
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+# Add script directory to the PATH
+export PATH="$SCRIPT_DIR:$PATH"
+
+get_didc_release(){
+  OS=$(uname)
+  ARCH=$(uname -m)
+  if [ "$OS" == "Darwin" ] && [ "$ARCH" == "arm64" ]; then
+    # Apple sillicon
+    wget -O didc https://github.com/dfinity/candid/releases/download/2023-07-25/didc-macos
```

Comment:
Maybe we can move the release directory (`https://github.com/dfinity/candid/releases/download/2023-07-25`) into a constant? It's used all over the place and this facilitate changing the release later.

Reply:
It is used only in two places in the link, so I do not believe it will be beneficial.